src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -127,6 +129,26 @@ public void setPerson(Person target, Person editedPerson) {
         addressBook.setPerson(target, editedPerson);
     }
 
```

Comment:
Since there will be Assessment, Tutorial, Person, will it be neater to separate the Assessment accessor methods from the others by using "//====Assessment Accessors==" etc just like that of Filtered Person's List?

Reply:
good idea, i'll add it in :)